bnd.bnd

```diff
@@ -1,40 +1,27 @@
+# make sure to not import classes which are not actually used from the embedded JARs
 Import-Package: !bsh, \
```

Comment:
@raducotescu Not sure who you came up with this list initially, but not importing javax.servlet.jsp seems like a mistake as the library is depending on it (and all distributions for sure export it)